### ISSUE TITLE
Error ('test_helper_blocksize.R:34:3'): blockSize reports block size in row-col order

### ISSUE THREAD

**Issue by barracuda156** [isciences/exactextractr#98]
```
R version 4.3.1 (2023-06-16) -- "Beagle Scouts"
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: powerpc-apple-darwin10.0.0d2 (32-bit)

> library(testthat)
> library(exactextractr)
The legacy packages maptools, rgdal, and rgeos, underpinning the sp package,
which was just loaded, will retire in October 2023.
Please refer to R-spatial evolution reports for details, especially
https://r-spatial.org/r/2023/05/15/evolution4.html.
It may be desirable to make the sf package available;
package maintainers should consider adding sf to Suggests:.
The sp package is now running under evolution status 2
     (status 2 uses the sf package in place of rgdal)
> 
> test_check("exactextractr")
[ FAIL 1 | WARN 1 | SKIP 0 | PASS 463 ]

══ Failed tests ════════════════════════════════════════════════════════════════
── Error ('test_helper_blocksize.R:34:3'): blockSize reports block size in row-col order ──
Error: [writeRaster] invalid driver
Backtrace:
    ▆
 1. ├─base::suppressWarnings(...) at test_helper_blocksize.R:34:2
 2. │ └─base::withCallingHandlers(...)
 3. ├─terra::writeRaster(...) at test_helper_blocksize.R:35:4
 4. └─terra::writeRaster(...)
 5.   └─terra (local) .local(x, filename, ...)
 6.     └─terra:::messages(x, "writeRaster")
 7.       └─terra:::error(f, x@ptr$getError())

[ FAIL 1 | WARN 1 | SKIP 0 | PASS 463 ]
Error: Test failures
Execution halted
```
Could this be fixed?

**Comment by dbaston** [comment]
Should be fixed with [dd56996](https://github.com/isciences/exactextractr/commit/dd5699629377088ffe9880432340e7d6ef838e0c)

**Comment by barracuda156** [comment]
@dbaston Thank you!
